skietbaan_fe/src/components/Register.js

```diff
@@ -167,8 +168,12 @@ class Register extends Component {
                 {invalidPasswordMessage}
               </FormGroup>
             </Col>
-            <Button onClick={this.Login} className={this.state.validForm ? "button-valid" : "button-invalid"} >Submit</Button>
+            <div className="button-container">
+            <Button onClick={this.Register} className={this.state.validForm ? "round-button" : "button-invalid round-button"} >Join</Button>
```

Comment:
you can multiline this

Reply:
<Button onClick={this.Register} 

className={this.state.validForm ? "round-button" : "button-invalid round-button"} >

Join

</Button>

Reply:
just apply correct indentation